src/features/navigation/RootNavigator/linking/getInitialUrl.ios.ts

```diff
@@ -6,7 +6,9 @@ import { BatchPush } from 'libs/react-native-batch'
 // firebase dynamic links work on iOS :
 // See: https://medium.com/tribalscale/working-with-react-navigation-v5-firebase-cloud-messaging-and-firebase-dynamic-links-f8580c3b83ad
 export async function getInitialURL(): Promise<string> {
-  // Little break for react-navigation loading
+  // When opening the app through an external link (without the app being in the background),
+  // the routes are not ready and the redirection is systematically done to home.
```

Comment:
the routes are not ready **and the user is systematically redirected to the homepage**.